src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -231,6 +221,28 @@ export default function ChartExpandedState({ asset }) {
     networks,
   } = useAdditionalAssetData(asset?.address, assetWithPrice?.price?.value);
 
+  // This one includes the original l2 address if exists
+  const ogAsset = useMemo(() => {
+    if (networks) {
+      let mappedNetworks = {};
+      Object.keys(networks).forEach(
```

Comment:
this was added to fix the supported networks mismatch between networks & implementations coming from the backend